### DISCUSSION TITLE
GitHub by itself jumps between last clicked comments in issue every 2-3 seconds

### DISCUSSION THREAD

**Discussion by ghost** [community/community#72771] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

- visit https://github.com/uBlockOrigin/uAssets/issues/19976#issuecomment-1783621437
- click "load more" if needed
- scroll a bit (issue may start right now - you will be scrolled back to comment you navigated to)
- click the hidden comment to display the content
- scroll past the hidden message
- click the timestamp in next message
- page now start jumping between current and previous (originally navigated to) comment repeatedly

This is on Firefox 121.0a1

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by wayosu** [comment]
Here are some steps to troubleshoot and resolve this issue:

**Check Browser Extensions:**
Disable any browser extensions or add-ons one by one to see if any of them are causing this behavior. Some extensions, particularly those designed to enhance GitHub or web browsing in general, might trigger this automatic scrolling behavior.

**Clear Browser Cache and Cookies:**
Sometimes, browser cache and cookies can cause unexpected behavior on websites. Clear your browser cache and cookies, then reload the GitHub page to see if the issue persists.

**Test in Different Browsers:**
Open the same GitHub issue in a different web browser to see if the automatic scrolling issue is specific to one browser. If it doesn't occur in another browser, the problem might be related to the browser you were using.

**Custom Scripts or User Styles:**
If you have any custom scripts or user styles applied to GitHub or your web browser (e.g., using a browser extension like Stylish), check if any of them are causing the issue. Temporarily disable these scripts or styles to test if they are responsible for the behavior.

**GitHub Feature or Bug:**
GitHub occasionally introduces new features or changes to its user interface. This behavior may be related to a specific GitHub feature or bug. Check the GitHub Help or Support forum for any announcements or discussions related to this issue.

**Update Browser and Extensions:**
Ensure that your web browser and all extensions are up to date. Sometimes, older browser versions or outdated extensions can exhibit unexpected behavior on websites.

**GitHub Settings:**
Check your GitHub settings for any configuration options that could be affecting comment navigation or scrolling behavior. GitHub allows you to customize your user settings, so review these settings to rule out any accidental changes.

**Comment by ghost** [comment, marked as answer]
Seems to be caused by one of the videos in the thread. Some exceptions when browser is trying to preload the content.
